docs/modern/PaginationContainer.md

```diff
@@ -85,7 +85,7 @@ class Feed extends React.Component {
           edge => <Story story={edge.node} key={edge.node.id} />
         )}
         <button
-          onPress={this._loadMore}
+          onPress={() => this._loadMore()}
```

Comment:
You could use property initializer syntax, but that's an early feature. This fix seems good here.